.pipelines/azdo-base-pipeline.yml

```diff
@@ -1,7 +1,17 @@
 steps:
-- script: |
-    ./lint-and-test.sh
-  displayName: 'Linting & unit tests'
+- task: PythonScript@0
+  inputs:
+    scriptSource: inline
+    script: set -eux
+            flake8 --output-file=lint-testresults.xml --format junit-xml
```

Comment:
[This fails](https://aidemos.visualstudio.com/MLOps/_build/results?buildId=3772&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=0d5f46f2-efd2-5fe7-3c1a-f6adc81b525b). Perhaps use the following which adds the | :
```
- script: |
   flake8 --output-file=$(Build.BinariesDirectory)/lint-testresults.xml --format junit-xml  
```